python/ray/autoscaler/local/config.py

```diff
@@ -1,2 +1,21 @@
+import logging
+
+logger = logging.getLogger(__name__)
+
+
 def bootstrap_local(config):
+    # Copy max_workers to config["provider"] in on-prem auto-management mode.
+    if "server_address" in config["provider"]:
+        config["provider"]["max_workers"] = config["max_workers"]
+
+        # Verify no autoscaling.
```

Comment:
It actually supports autoscaling now that we have a coordinating server right?

Reply:
I thought we agreed in the design doc not to support autoscaling. If we move to work with the server doing something similar to the cloud gateway, then we can support autoscaling, the drawback is that create node might fail if in between two calls of non_terminated_nodes and create_node another person took the node.